.storybook/main.ts

```diff
@@ -1,25 +1,25 @@
-import { StorybookConfig } from '@storybook/react-webpack5';
-
-const config: StorybookConfig =  {
+import type { StorybookConfig } from "@storybook/react-vite";
+const config: StorybookConfig = {
   stories: [
-    "../stories/**/*.stories.js",
+    "../src/stories/**/*.stories.jsx",
```

Comment:
* Adjusted folder due to moving the files
* Switched those files to JSX so that Storybook know its a JSX file without extra configuration (can impact which indexer is used and how the file is transformed)